pkg/agent/datamodel/const.go

```diff
@@ -104,3 +104,7 @@ const (
 	// when KubeletDiskType == TempDisk.
 	TempDiskContainerDataDir = "/mnt/aks/containers"
 )
+const (
+	// InstanceErrorCode is used to parse instance error code for vmss or vmas
+	InstanceErrorCode = "InstanceErrorCode"
+)
```

Comment:
I feel we don't need to expose this in public contract. Just inline the value if needed